File: pylabs/lab1.py
from math import *

a = float(input("Введите a:"))
x = float(input("Введите x:"))
g = - (16 * a ** 2 + 24 * a * x - 27 * x ** 2) / (45 * a ** 2 - 29 * a * x + 4 * x ** 2)
print("G =", g, end="")

a = float(input("Введите a:"))
x = float(input("Введите x:"))
f = -atan(10 * a ** 2 + 13 * a * x - 30 * x ** 2)
print("F =", f, end="")

a = float(input("ВВедите a:"))
x = float(input("Введите x:"))
y = log(2 * a ** 2 + 19 * a * x + 9 * x ** 2 + 1) / log(10)
print("Y =", y, end="")

File: clabs/lab1.c
#include <stdio.h>
#include <math.h>

int main(void) 
{
    float a, x, g, f, y;
    printf("Vvedite a: ");
    scanf("%f", &a);
    printf("Vvedite x: ");
    scanf("%f", &x);
    g = - (16 * a * a + 24 * a * x - 27 * x * x) / (45 * a * a - 29 * a * x + 4 * x * x);
    printf("G=%f\n\n", g);

    printf("Vvedite a: ");
    scanf("%f", &a);
    printf("Vvedite x: ");
    scanf("%f", &x);
    f = -atan(10 * a * a + 13 * a * a - 30 * x * x);
    printf("F=%f\n\n", f);

    printf("Vvedite a: ");
    scanf("%f", &a);
    printf("Vvedite x: ");
    scanf("%f", &x);
    y = log(2 * a * a + 19 * a * x + 9 * x * x + 1) / log(10);
    printf("Y=%f\n\n", y);
}
